.github/workflows/CoverageCollection.yml

```diff
@@ -117,15 +122,19 @@ jobs:
       - name: Collect coverage
         continue-on-error: true
         run: |
-          [ 'x${{ inputs.coverage_config }}' != 'x' ] && PYCOV_ARGS='--cov-config=${{ inputs.coverage_config }}' || unset PYCOV_ARGS
-          python -m pytest -rA --cov=. $PYCOV_ARGS ${{ inputs.unittest_directory }} --color=yes
+          RELDIR="$(realpath --relative-to=${{ inputs.test_directory }} .)"
```

Comment:
Why do we need the coverage config to be relative to the workdir? We can get the absolute location (through `pwd`), before `cd ${{ inputs.testdirectory }}`.

Reply:
Both solution require a similar amount of extra scripting / variables. Coverage is not executed on macOS, so no problems with `realpath`.